src/sumBy.test.ts

```diff
@@ -17,6 +17,16 @@ describe("data first", () => {
       ),
     ).toEqual(25);
   });
+
+  test("works with bigint", () => {
+    expect(
+      sumBy([{ a: 1n }, { a: 2n }, { a: 4n }, { a: 5n }, { a: 3n }], prop("a")),
+    ).toEqual(15n);
+  });
+
+  it("should return 0 for an empty array", () => {
+    expect(sumBy([] as Array<{ a: bigint }>, prop("a"))).toBe(0);
+  });
```

Comment:
This is not a runtime test; it shouldn't be in this file.
```suggestion
```

Reply:
the `sumBy` test lacks the empty array case. So I removed the type and kept the test to ensure it returns 0 when the parameter is an empty array